Frontend/javascript/geoCoding.js

```diff
@@ -0,0 +1,27 @@
+function countriesCallBackEnd(){
+    $.ajax({
+        url: 'http://127.0.0.1:8082/GeoJson',
+        data: "<?xml version='1.0' encoding='UTF-8'?>"+
+        "<Request>"+
+        "<requestName>ISSDB<requestName>"+
```

Comment:
Wrong requestName! Doesn't matter for our BE, but the requestname is still the wrong one. I also hope the data of this call is cached. Because the call takes up to 2+ seconds. Also this solution is very inperformant because you'll get so much data and you only need 1% of them. Nevertheless I'll approve it, but I'll add an Issue labeled as enhancement for this.

Reply:
But please change the Requestname